metadata-ingestion/setup.py

```diff
@@ -311,6 +311,7 @@
     | classification_lib,
     "clickhouse": sql_common | clickhouse_common,
     "clickhouse-usage": sql_common | usage_common | clickhouse_common,
+    "cockroachdb": sql_common | {"psycopg2-binary", "GeoAlchemy2", "sqlalchemy-cockroachdb<2.0.0"},
```

Comment:
Let's extract out a `postgres_common` and use that here and for postgres - that way it's a bit more clear that this is built on top of postgres

Reply:
@hsheth2 done